modules/store.js

```diff
@@ -0,0 +1,33 @@
+/* eslint-disable linebreak-style */
+/* eslint-disable no-use-before-define */
+/* eslint-disable max-classes-per-file */
+/* eslint-disable no-trailing-spaces */
+/* eslint-disable lines-between-class-members */
+
```

Comment:
- [ ] Kindly avoid disabling linters. Linters help us write standardized code. Enable these linters and resolve any resulting issues.